src/components/SelectTime/SelectRegion.jsx

```diff
@@ -0,0 +1,68 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+
+const SelectRegion = () => {
+  const [checkedRegions, setCheckedRegions] = useState([]);
+
+  const handleOnClick = region => {
+    setCheckedRegions([region]);
+  };
+
+  const RegionList = ['홍대', '청담씨네시티', '목동', '피카디리1958', '신촌아트레온'];
+
+  return (
+    <>
+      <St.SelectRegion>
+        {RegionList.map(region => (
+          <St.RegionButton
+            key={region}
+            onClick={() => handleOnClick(region)}
+            isChecked={checkedRegions.includes(region)}
+          >
+            {region}
+          </St.RegionButton>
+        ))}
+      </St.SelectRegion>
+    </>
+  );
+};
+
+export default SelectRegion;
+
+const St = {
+  SelectRegion: styled.section`
+    width: 37.5rem;
+    display: flex;
+    align-items: center;
+    gap: 0.8rem;
+    padding: 1.6rem;
+    gap: 0.8rem;
+    overflow: auto;
+    white-space: nowrap;
+    overflow: scroll;
+
+    &::-webkit-scrollbar {
+      display: none;
+    }
+  `,
+
+  RegionButton: styled.button`
```

Comment:
p1) typescript에서는 props 내려줄 때 타입을 지정해줘야 해요!

```suggestion
  RegionButton: styled.button<{ $isClicked : boolean}>`
```

이렇게 해주면 아마 에러 뜨던거 해결될것 같습니댜 !!